README.md

```diff
@@ -905,6 +920,20 @@ turbo price --value 1024 --type bytes
 turbo price --value 1.1 --type arweave
 ```
 
+##### `token-price`
+
+Get the current price from the Turbo Payment Service, denominated in the specified token, for uploading a specified number of bytes to Turbo.
+
+Command Options:
+
+- `--value <value>` - Byte value to get the token price for
```

Comment:
Is there a reason this parameter needs to be named `value`? Why not `byteCount`?

Reply:
171eadd